src/Controller/CoversController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace eLife\Journal\Controller;
+
+use eLife\Patterns\ViewModel\Button;
+use eLife\Journal\ViewModel\Paragraph;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use eLife\Patterns\ViewModel\Image;
+use eLife\Patterns\ViewModel\Picture;
+use GuzzleHttp\Client;
+
+
+final class CoversController extends Controller
```

Comment:
We'll need a different name, 'cover' is already a different concept.

Reply:
@thewilkybarkid will change